pkg/proto/hapi/chart/metadata.pb.go

```diff
@@ -74,7 +74,7 @@ type Metadata struct {
 	// The condition to check to enable chart
 	Condition string `protobuf:"bytes,11,opt,name=condition" json:"condition,omitempty"`
 	// The tags to check to enable chart
-	Tags []string `protobuf:"bytes,12,opt,name=tags" json:"tags,omitempty"`
```

Comment:
It was a change that resulted from running `make protoc` after I upgraded to 3.2. 
I took it out of this commit though. Thanks.